javascriptlib/src/mill/javascriptlib/PublishModule.scala

```diff
@@ -189,7 +180,12 @@ trait PublishModule extends TypeScriptModule {
     // bundled code for publishing
     val bundled = bundle().path / os.up
 
-    os.walk(bundled, skip = p => p.last == "node_modules" || p.last == "package-lock.json")
+    os.walk(
+      bundled,
+      skip = p =>
+        p.last == "node_modules" ||
+          p.last == "package-lock.json"
+    )
```

Comment:
This "walk folder ignoring node_modules and pacakge-lock.json" logic is common enough it should be moved into a helper. Do we really need to skip those two filenames in all those callsites